docs/developer_guide/getting_started.md

```diff
@@ -1,67 +1,46 @@
 
-![Monero-javascript hierarchy](img/architecture.png?raw=true)*Monero-javascript can communicate through three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
+![Monero-javascript hierarchy](img/paste.png?raw=true)*Monero-javascript manipulates XMR and communicates with the blockchain via three channels: RPC wallet servers, RPC daemon servers, and Wasm wallets.*  
 
-# Initial Setup
+## Initial Setup
 
-Skip ahead to [Create a new Node.js project](todo) if you already have Node.js and npm installed. 
+Skip ahead to [Create a new Node.js project](#create-a-new-node.js-project) if you already have Node.js and npm installed.
 
-## Install Node.js and npm
-You need to install node.js and the node package manager (npm) to obtain and use the monero-javascript library. 
-
-### Windows
-1. [Download the Node.js Windows installer](https://nodejs.org/en/download/) from the Node.js website.
-2. Open the installer.
-3. Click “next”.
-4. Click the checkbox next to “I accept the terms in this license agreement.” and click “next”.
-5. Click “next” to install Node.js to the default directory.
-6. Make sure that npm is listed as one of the installation packages, then click “next”.
-7. Click “next”.
-8. Click “Install”.
-9. Click “Finish” after the installation process completes.
-
-### Linux
-  #### Debian (Ubuntu, etc.)
-  1. Install node.js:
-    `$ sudo apt-get install nodejs`
-  2. Install npm:
-    `$ sudo apt-get install npm`
-  #### Fedora
-  1. Install node.js:
-    `$ sudo dnf install nodejs`
-  2. Install npm:
-    `$ sudo dnf install npm`
+### Install Node.js and npm
+You need to install node.js and the node package manager (npm) to obtain and use the monero-javascript library. See the ["Node.js and npm"](https://github.com/monero-ecosystem/monero-javascript/blob/master/docs/developer_guide/installing_prerequisite_software.md#nodejs-and-npm) section of the prerequisite installation guide for step-by-step instructions for downloading and installing Node.js and npm.
 
 ## Create a new Node.js project
 
 1. Create and enter a new directory to hold the project:
+  ### Linux/Mac
+  1. `mkdir ~/offline_wallet_generator`
+  2. `cd ~/offline_wallet_generator`
+
   ### Windows
   1. `mkdir C:\offline_wallet_generator`
   2. `cd C:\offline_wallet_generator`
 
-  ### Linux
-  1. `mkdir ~/offline_wallet_generator`
-  2. `cd ~/offline_wallet_generator`
-  
 2. Create the new project:
   `npm init`
-  npm will ask you to enter information into a number of fields. Press enter at each prompt to use the default (or empty) value.
+  The npm setup utility will ask you for some information. Press enter at each prompt to use the default value.
 
 3. Add the monero-javascript library to the offline_wallet_generator package:
   `npm install monero-javascript`
```

Comment:
Shouldn't that be `npm install --save monero-javascript` ?

Reply:
https://stackoverflow.com/questions/19578796/what-is-the-save-option-for-npm-install

"As of npm 5.0.0, installed modules are added as a dependency by default, so the --save option is no longer needed. The other save options still exist and are listed in the documentation for npm install."

Is there any reason to expect anyone would be using a version of npm pre-v6?

Reply:
Doing a `npm i -g npm` will get you a `6.14.5`

Is there any reason for not being specific about what you mean to do?
This is not a breaking issue. Assuming everybody is on your page is fine with me.

Reply:
Trasher, I'm not exactly an expert with node or npm, and I'm a little confused by your meaning here. I'm going to fix this to use the --save option since it should ensure backwards compatibility and dos not interfere with functionality in npm versions >= 5.

I assume you mean that by omitting the --save, I'm not being specific about what I want to do with the command. I honestly didn't know there WAS a save option until your comment, or that it was ever necessary in any version of npm to begin with. I just thought "npm install _package_" is the way you install npm packages period.

Obviously I'm in the process of learning all of this.